apps/src/lib/node/ledger/mod.rs

```diff
@@ -442,15 +453,17 @@ fn start_abci_broadcaster_shell(
     // Construct our ABCI application.
     let tendermint_mode = config.tendermint.tendermint_mode.clone();
     let ledger_address = config.shell.ledger_address;
-    let (shell, abci_service) = AbcippShim::new(
+    let event_log_sender = Some(event_log_sender);
```

Comment:
Shouldn't we pass in a `None` value if we are not a validator?

Reply:
when I implement the event log, I'll update this. this value is pretty much just a placeholder, for the ledger to compile. there's a TODO item above the binding of `event_log_sender` explaining this